modules/dnn/src/darknet/darknet_importer.cpp

```diff
@@ -179,7 +185,32 @@ class DarknetImporter
 
 Net readNetFromDarknet(const String &cfgFile, const String &darknetModel /*= String()*/)
 {
-    DarknetImporter darknetImporter(cfgFile.c_str(), darknetModel.c_str());
+    std::ifstream cfgStream(cfgFile.c_str());
+    CV_Assert(cfgStream.is_open());
```

Comment:
There were assertions include paths to files,
```
CV_Error(cv::Error::StsParseError, "Failed to parse NetParameter file: " + std::string(cfgFile));
```
I think it's useful to keep some kind of error message that prints actual paths.

Reply:
I think so too. I added it.